Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -347,6 +348,38 @@ lemma gaussianReal_mul_const {X : Ω → ℝ} (hX : Measure.map X ℙ = gaussian
 
 end Transformations
 
+open Measurable Real
+
+theorem mgf_gaussian {Ω : Type} {m : MeasurableSpace Ω} {μ : Measure Ω}
+    (X: Ω → ℝ) (hXm: Measurable X) (hX: (Measure.map X μ) = (gaussianReal 0 1)) (t : ℝ) :
+    mgf X μ t = exp (t ^ 2 / 2) := calc
+  mgf X μ t = (Measure.map X μ)[fun x => exp (t * x)] := by
+    have : AEStronglyMeasurable (fun x ↦ exp (t * x)) (Measure.map X μ) :=
+      AEMeasurable.aestronglyMeasurable (aemeasurable
+      (measurable_exp.comp (Continuous.measurable (continuous_const.mul continuous_id))))
```

Comment:
I won't insist that you use dot notation everywhere (it's not required by the style guide), but here is an example in case you find it nicer:
```suggestion
      (measurable_exp.comp (continuous_const.mul continuous_id).measurable))
```